lib/kitchen/driver/ec2.rb

```diff
@@ -97,14 +98,23 @@ class Ec2 < Kitchen::Driver::SSHBase
         end
       end
 
+      def self.iam_creds
+        @iam_creds ||= begin
+         fetch_credentials(use_iam_profile:true)
+       rescue
+         debug("fetch_credentials failed with exception #{e.message}:#{e.backtrace.join("\n")}")
```

Comment:
The var `e` doesn't exist here because you haven't declared it.  The rescue line should look like `rescue RuntimeError => e`